components/cli/cli/file_system.go

```diff
@@ -25,15 +25,23 @@ import (
 )
 
 const celleryHome = ".cellery"
+const celleryInstallationPathMac = "/Library/Cellery"
+const celleryInstallationPathUbuntu = "/usr/share/cellery"
 
 type FileSystemManager interface {
 	CurrentDir() string
-	UserHome() (string, error)
+	UserHome() string
+	TempDir() string
 	Repository() string
+	RemoveAll(path string) error
```

Comment:
Why do we have RemoveAll() method here? do we need to mock it? Normally for a unit test we do not need to mock such a method.

Reply:
In "cellery delete" command, the images are getting deleted. In the unit tests we mock the repo (test data) and provide mock images. The purpose of having a RemoveAll interface function is so that during unit tests, the mock images do not get deleted.
Perhaps copy the test data to a temp dir before the unit tests so that mocked images do not get deleted? WDYT.

Reply:
This is not correct IMO. Deleting files is not something we need to mock. Keep a sample image in your testdata directory, copy it to a temp repo location from the test and then assert that its properly getting deleted.

Reply:
Removed RemoveAll interface function. Used a copy of mock repo to a temp location in the unit test.